monkey/infection_monkey/network/postgresql_fingerprint.py

```diff
@@ -0,0 +1,102 @@
+import logging
+
+import psycopg2
+
+from infection_monkey.network.HostFinger import HostFinger
+
+LOG = logging.getLogger(__name__)
+
+
+class PostgreSQLFinger(HostFinger):
+    """
+    Fingerprints PostgreSQL databases, only on port 5432
+    """
+    # Class related consts
+    _SCANNED_SERVICE = 'PostgreSQL'
+    POSTGRESQL_DEFAULT_PORT = 5432
+    CREDS = {'username': "monkeySaysHello",
```

Comment:
For these strings I think we should use `ID_STRING` from model. If we unify the strings monkey uses, it will be easier for users to identify monkey activity in the logs.